libyang/schema.py

```diff
@@ -790,16 +775,16 @@ class SNode:
         INPUT: "input",
         OUTPUT: "output",
         NOTIF: "notification",
+        ANYXML: "anyxml",
+        ANYDATA: "anydata",
     }
 
     def __init__(self, context: "libyang.Context", cdata):
         self.context = context
-        self.cdata = cdata  # C type: "struct lys_node *"
-
-    @property
-    def _node(self):
-        deprecated("_node", "cdata", "2.0.0")
-        return self.cdata
+        self.cdata = cdata  # C type: "struct lysc_node *"
+        self.cdata_parsed = None
+        if self.cdata.priv != ffi.NULL:
```

Comment:
Hello, why do you put this condition here ? cdata.priv is always NULL

Reply:
And `self.cdata_parsed` is used at several place without checking if it is set so it creates lot of errors.

Reply:
> cdata.priv is always NULL

Flag [LY_CTX_SET_PRIV_PARSED](https://netopeer.liberouter.org/doc/libyang/devel/html/group__contextoptions.html#ga7883206423631283065208634d57e056) is set in context init method. And it is necessary for easy access to both parsed and compiled variants of a schema node. So `self.cdata_parsed` should be always also set.